src/impl_aarch64__base_implementation.inl

```diff
@@ -0,0 +1,100 @@
+// Copyright 2017 Google LLC
```

Comment:
`impl_aarch64__base_implementation` implies that this is a general solution in which there may be a compiler dependency and there should be no OS dependency, like in `impl_x86__base_implementation`. Could you rename this file?

Reply:
I removed all the OS dep from that file and retained the file name.